registry.k8s.io/images/k8s-staging-sig-storage/generate.sh

```diff
@@ -18,10 +18,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-readonly repo="gcr.io/k8s-staging-sig-storage"
+readonly repo="us-central1-docker.pkg.dev/k8s-staging-sig-storage"
```

Comment:
This gives me 404 today (`gcloud container images list-tags us-central1-docker.pkg.dev/k8s-staging-sig-storage/csi-attacher`).

How can we confirm it's the right address?

Reply:
We have not pushed any images to the new staging repo yet, so I think you will get a 404 if you are looking specifically for csi-attacher images.